app/schemas/models/prepaid.schema.coffee

```diff
@@ -19,6 +20,7 @@ PrepaidSchema = c.object({title: 'Prepaid', required: ['type']}, {
     gems: {type: 'number', description: 'Overwrites the number of gems given by Product when the prepaid is redeemed'}
     expires: c.stringDate({description: 'Date the prepaid expires and cannot be redeemed'})
   endDate: c.stringDate()
+  duration: {type: 'number', description: 'Prepaid valid duration, in milliseconds'}
```

Comment:
I think the duration here might be better in days? We usually measure API license usage in days, for example.  Is there a reason to use milliseconds?

Reply:
emmm. no special reason to use milliseconds , in frontend we set this thing as days and i multiple the (3600*24*1000), so that when we calculate it we can just use `new Date() + duration` , but of casue we can save days i think